src/blocks/CardLayout/withCardLayoutItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {Fragment} from 'react';
+import {useCardLayoutContext} from './CardLayoutContext';
+import {block} from '../../utils';
+import {Col} from '../../grid';
+import {ItemWrapper} from '../../context/innerContext';
+
+const b = block('card-layout-block');
```

Comment:
Didn't find usage

Reply:
Please, have a look at line 13

Reply:
It is confusing that we copy the block-name from other component and use only as a child. Maybe CardLayoutItem should be placed to CardLayoutBlock then and use here as import?